packages/SimPKPD/src/package.ts

```diff
@@ -2,80 +2,75 @@
 
-export const _package = new DG.Package();
-
-//name: Simulation PKPD
-//tags: app
-export async function sim() {
-  const dosage = ui.floatInput('Dosage, um', 1000, () => recalculate());
-  const doseInterval = ui.floatInput('Dose intrval, h', 12, () => recalculate());
-  const compartments = ui.choiceInput('Model ', '2 compartment PK', ['2 compartment PK', '1 compartment PK'], () => recalculate());
-  const clearance = ui.floatInput('Clearance', 2, () => recalculate());
-  const rateConstant = ui.floatInput('Rate', 0.3, () => recalculate());
-  const centralV = ui.floatInput('Central volume', 4, () => recalculate());
-  const perV = ui.floatInput('Peripheral volume', 30, () => recalculate());
-  const interRate = ui.floatInput('Intercompartmental rate', 1, () => recalculate());
-  const effRate = ui.floatInput('Effective rate', 0.2, () => recalculate());
-  const effect = ui.floatInput('EC50', 8, () => recalculate());
-
-  let t = await simulate(dosage.value!, doseInterval.value!, compartments.value!,
-  clearance.value!, rateConstant.value!, centralV.value!,
-  perV.value!, interRate.value!, effRate.value!, effect.value!);
-
-  let lc = DG.Viewer.lineChart(t);
+import { initSolvers } from '../wasm/solving-tools';
+import { showHelpPanel } from '../wasm/help-panel';
+import { simPKPD } from './pk-pd-tools';
 
-  let recalculate = async (): Promise<void> => {
-    t = await simulate(
-      dosage.value!, doseInterval.value!, compartments.value!,
-      clearance.value!, rateConstant.value!, centralV.value!,
-      perV.value!, interRate.value!, effRate.value!, effect.value!);
+export const _package = new DG.Package();
 
-    lc.dataFrame = t;
-  }
+//tags: init
+export async function init(): Promise<void> {
+  await initSolvers();
+}
 
-  const v = grok.shell.newView('SimPKPD', [
-    ui.panel([
-      ui.div([
-        ui.div([
-          ui.divH([ui.h1('Inputs')]),
-          ui.divV([
-            dosage,
-            doseInterval,
-            compartments,
-            clearance,
-            rateConstant,
-            centralV,
-            perV,
-            interRate,
-            effRate,
-            effect
-          ], 'ui-form'),
-        ], 'ui-form'),
-      ], 'ui-form'),
-    ])
-  ]);
-  v.box = true;
+//name: PK-PD
+//description: Two-compartment pharmacokinetic-pharmacodynamic (PK-PD) simulation.
+//tags: model
+//input: double dose = 10000.0 {units: um; caption: dose; category: Dosing} [Dosage.]
+//input: int dosesCount = 10 {caption: count; category: Dosing; min: 1; max: 1000} [Number of doses.]
+//input: double doseInterval = 12 {units: h; caption: interval; category: Dosing; min: 0; max: 100} [Dosing interval.]
+//input: double _KAVal = 0.3 {units: ; caption: rate constant; category: PK parameters}
+//input: double _CLVal = 2.0 {units: ; caption: clearance; category: PK parameters}
+//input: double _V2Val = 4.0 {units: ; caption: central volume; category: PK parameters} [Central compartment volume.]
```

Comment:
would be great to add reasonable min/max to parameters, so that we can use clickers and sliders in inputs

Reply:
Done